bot.py

```diff
@@ -68,14 +68,22 @@ async def on_command_error(ctx, error):
     msg = "".join(traceback.format_exception(type(error), error, error.__traceback__))
     logger.error(msg)
     logger.error(f'Error caused by user {ctx.author} running the command {ctx.message.content}')
-
-    connection = sqlite3.connect('db/kolulu.db')
-    with closing(connection) as db:
-        statement = 'INSERT INTO errors (user_id, stacktrace, command) VALUES (?, ?, ?)'
-        db.cursor().execute(statement, (ctx.author.id, msg, ctx.message.content))
-        db.commit()
-
-    await ctx.send(f'Something went wrong.\nError: {error}')
+    try:
+        connection = sqlite3.connect('db/kolulu.db')
+        guildId = ctx.guild.id
+        with closing(connection) as db:
+            statement = 'INSERT INTO errors (user_id, stacktrace, command) VALUES (?, ?, ?)'
+            db.cursor().execute(statement, (ctx.author.id, msg, ctx.message.content))
+            db.commit()
+            statement = 'SELECT silent FROM silence WHERE server_id = ?'
+            cursor = db.cursor()
+            cursor.execute(statement, (guildId,))
+            result = cursor.fetchone()
+            if result[0]==0:
+                await ctx.send(f'Error: {error}')
+            db.commit()
+    except:
+        await ctx.send(f'Error: {error}')
```

Comment:
We still want to log the error into the db even in dm. Maybe add a boolean that will get switched during the except block? And only do the silent check if the boolean doesn't get switched